docs/modules/ROOT/pages/type-definitions/access-control.adoc

```diff
@@ -109,3 +109,100 @@ This field will only feature in input types, and will not be available for query
 """Instructs @neo4j/graphql to only include a field in the generated input types for the object type within which the directive is applied, but exclude it from the object type itself."""
 directive @writeonly on FIELD_DEFINITION
 ----
+
+[[type-definitions-access-control-selectable]]
+== `@selectable`
+
+This directive controls if the field will be available on queries and aggregations. 
+
+It has two arguments:
+
+* onRead: If disabled, this field will not be available on queries and subscriptions.
+* onAggregation: If disabled, aggregations will not be available for this field
+
+
+=== Definition
+
+[source, graphql, indent=0]
+----
+"""Instructs @neo4j/graphql to generate this field for selectable fields."""
+directive @selectable(onRead: Boolean! = true, onAggregate: Boolean! = true) on FIELD_DEFINITION
+----
+
+=== Usage
+
+With the following definition:
+
+[source, graphql, indent=0]
+----
+type Movie {
+    title: String!
+    description: String @selectable(onRead: false, onAggregate: true)
+}
+----
+
+The type `Movie` in the resulting schema will look:
+
+[source, graphql, indent=0]
+----
+type Movie {
+    title: String!
+}
+----
+
+This means that descriptions cannot be queries, neither on top or nested levels. Aggregations, however, are available for both:
+
+[source, graphql, indent=0]
+----
+type MovieAggregateSelection {
+    count: Int!
+    description: StringAggregateSelectionNullable!
+    title: StringAggregateSelectionNonNullable!
+}
+----
+
+[[type-definitions-access-control-settable]]
+== `@settable`
+
+This directive controls if the input field will be available on creation and update mutations.
+
+It has two arguments:
+
+* onCreate: If disabled, this field will not be available to be set on creation.
+* onCreate: If disabled, this field will not be available to be set on update.
```

Comment:
This is the same argument than the one above but it has a different description? I think it was supposed to be onUpdate?

```suggestion
* *onUpdate*: If disabled, this field will not be available to be set on update.
```

Reply:
Yeah, you are totally right, thanks :+1: